sklearnex/_config.py

```diff
@@ -76,6 +96,28 @@ def set_config(target_offload=None, allow_fallback_to_host=None, **sklearn_confi
         local_config["target_offload"] = target_offload
     if allow_fallback_to_host is not None:
         local_config["allow_fallback_to_host"] = allow_fallback_to_host
+    if _is_dpc_backend and gpu_blas_compute_mode is not None:
+        if gpu_blas_compute_mode != "standard":
+            try:
+                inp = (
+                    gpu_blas_compute_mode.split(",")
+                    if type(gpu_blas_compute_mode) is str
+                    else gpu_blas_compute_mode
+                )
+                gpu_blas_compute_mode = ",".join(
+                    [ComputeMode.__members__[i.lower()].name for i in inp]
+                )
+                local_config["gpu_blas_compute_mode"] = gpu_blas_compute_mode
+                os.environ["DAL_BLAS_COMPUTE_MODE"] = gpu_blas_compute_mode.upper()
+
+            except (KeyError, AttributeError) as e:
+                raise ValueError(
+                    f"'{e.args[0]}' is not a supported gpu_blas_compute_mode"
```

Comment:
1) Looks too compolex to be a single function or an isolated piece of code
2) I would recommend to actually use a map:
```python
@lru_cache
def get_compute_mode_map():
    members = ComputeMode.__members__ 
    return { m.name.lower() : m for m in members }

def get_raw_compute_mode():
    ...

def set_raw_compute_mode(cm):
    ...
```